docs/_docs/2_4_configurations.md

```diff
@@ -510,3 +510,28 @@ Property: `hoodie.memory.compaction.fraction` <br/>
 #### withWriteStatusFailureFraction(failureFraction = 0.1) {#withWriteStatusFailureFraction}
 Property: `hoodie.memory.writestatus.failure.fraction` <br/>
 <span style="color:grey">This property controls what fraction of the failed record, exceptions we report back to driver</span>
+
+#### Write commit callback configs
+Controls callback behavior on write commit
+[withCallbackConfig](#withCallbackConfig) (HoodieWriteCommitCallbackConfig) <br/>
+<span style="color:grey">Callback related configs</span>
+
+##### writeCommitCallbackOn(callbackOn = false) {#writeCommitCallbackOn} 
+Property: `hoodie.write.commit.callback.on` <br/>
+<span style="color:grey">Turn callback on/off. off by default.</span>
+
+##### withCallbackClass(callbackClass) {#withCallbackClass} 
+Property: `hoodie.write.commit.callback.class` <br/>
+<span style="color:grey">Full path of user-defined callback class and must be a subclass of HoodieWriteCommitCallback class, org.apache.hudi.callback.impl.HoodieWriteCommitHttpCallback by default</span>
+
+##### withCallbackHttpUrl(url) {#withCallbackHttpUrl} 
```

Comment:
would add `#### HoodieWriteCommitHttpCallback` before the following configs since the configs belong to HoodieWriteCommitHttpCallback

Reply:
> would add `#### HoodieWriteCommitHttpCallback` before the following configs since the configs belong to HoodieWriteCommitHttpCallback

yes, my mistake.